src/store/actions/SovereignVenue.ts

```diff
@@ -45,7 +51,13 @@ export const setSovereignVenue = (
   payload: { sovereignVenue },
 });
 
+export const resetSovereignVenue = (): ResetSovereignVenueAction => ({
+  type: SovereignVenueActionTypes.RESET_SOVEREIGN_VENUE,
+  payload: {},
```

Comment:
This object has no purpose but I suppose payload is a required field. Can you check if you can make it optional somehow?